Error: Cannot find module 'detab'

<!-- **IMPORTANT!**
Before reporting a bug, please make sure that you have read through our documentation and you think your problem is indeed an issue related to our module. -->

### Version
@nuxt/content: 1.7.1
nuxt: 2.14.4

### Steps to reproduce
1. Create project with nuxt 2.14.4 and content 1.7.1.
2. Run `yarn nuxt`

### What is Expected?
No error

### What is actually happening?
Getting the following error:
```
Nuxt Fatal Error  
           
Error: Cannot find module 'detab'  
Require stack:  

- /sandbox/node_modules/@nuxt/content/parsers/markdown/handlers/code.js  
- /sandbox/node_modules/@nuxt/content/parsers/markdown/handlers/index.js 
- /sandbox/node_modules/@nuxt/content/parsers/markdown/index.js  
- /sandbox/node_modules/@nuxt/content/parsers/index.js  
- /sandbox/node_modules/@nuxt/content/lib/database.js  
- /sandbox/node_modules/@nuxt/content/lib/index.js  
- /sandbox/node_modules/@nuxt/core/dist/core.js  
- /sandbox/node_modules/@nuxt/cli/dist/cli-index.js  
- /sandbox/node_modules/@nuxt/cli/dist/cli.js  
- /sandbox/node_modules/nuxt/bin/nuxt.js     
```

--- a/packages/content/package.json
+++ b/packages/content/package.json
@@ -22,6 +22,7 @@
     "consola": "^2.15.0",
     "csvtojson": "^2.0.10",
     "defu": "^3.1.0",
+    "detab": "^2.0.3",
     "escape-html": "^1.0.3",
     "graceful-fs": "^4.2.4",
     "gray-matter": "^4.0.2",

